packages/cypress/src/generators/cypress-project/cypress-project.ts

```diff
@@ -47,6 +50,25 @@ function createFiles(tree: Tree, options: CypressProjectSchema) {
   const cypressFiles =
     cypressVersion && cypressVersion < 10 ? 'v9-and-under' : 'v10-and-after';
 
+  // I know this is sort of messy, I'll fix it
+  // It's mostly for POC now
+  const projectConfig = readProjectConfiguration(tree, options.project);
```

Comment:
--project isn't always guaranteed to be passed in. it can be omitted if the --base-url is passed in.